docs/versions/index.md

```diff
@@ -0,0 +1,30 @@
+# Versions
+
+!!! info inline end "Version 21.0 is current"
+
+    Read the [release notes for the current version](21.0).
+
+
+The primary mission of HGVS Nomenclature is to facilitate reliable communication of sequence variants, which requires that the HGVS Nomenclature is stable. Nonetheless, modifications will be required from time to time in order to address new scientific needs, resolve inconsistencies, or clarify conventions.  A key goal of the [HVNC](/hvnc/) is to manage such changes in a way that balances the needs of progress and reliable data sharing, and to communicate the changes to the community clearly.
+
+The HGVS Nomenclature uses concepts from [Semantic Versioning](https://semver.org/) to version releases. Each release version will consist of 3 numbers separated by dots in the format x.y.z with the following meanings:
+
+- The "patch" version (z) is reserved for changes that fix or clarify intentions. This is akin to a bug fix.
+- The "minor" version (y) will be used for new features that are consistent with prior versions but should not break them.  For example, adoption of a community proposal that added a new feature would trigger an increase in the minor version.
+- The "major" version (x) will be used when changes are incompatible with existing conventions, and perhaps for other changes that are deemed to be significant. Major changes should be rare. Wherever possible, the HVNC will strive to ensure that backward incompatibilities result in obvious failures (e.g., falure to parse at all) rather than subtle errors that are difficult to catch.
```

Comment:
What I'm trying to say is that backwards incompatibilities should be rare, and that when they are necessary, we will try to make the failure mode of the incompatibility obvious rather than silent.

For example, suppose we decided to use zero-based numbering. It would be awful to just say that in version 22 that c is zero based because lots of things might parse (and even be semantically plausible). Instead, we might introduce a new t type that is zero based and deprecate c.

So, to be clear, I'm not talking about specific software, but instead our expectation about consequences of changes.

Reply:
Good catch. Changed.

Reply:
Clear, thanks!
How about: "(...) ensure that backward incompatible changes result in obvious differences between variant descriptions based on the previous and current version of the nomenclature, rather than subtle differences that are difficult to catch."

Reply:
I think I see what you're getting at regarding the implication of some sort of test. How about this?

Wherever possible, the HVNC will strive to ensure that incompatible differences between versions are likely to (or can be made to) result in obvious failures when implemented in software (e.g., failure to parse at all) rather than subtle errors that are difficult to catch or might be overlooked.

Reply:
Looks great!